.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+---
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Why not daily?

Reply:
there are lots of dependencies, not sure how how often updates are published... we might want to change this later, when we are confident it is useful

Reply:
Sure, makesense. :+1:

Reply:
its good to weekly as it can pull the consolidated changes together for few at time.. daily looks like a an overkill :)

Reply:
I just want to avoid the need to trigger this manually on a release day, if that makesense.